src/asn1.mli

```diff
@@ -238,6 +278,14 @@ sig
   val dh_grammar : DH.Params.t Asn.t
 end
 
+module CVC :
```

Comment:
This sort of API is only useful if you don't know in advance what kind of key you'll parse (ex: in certificates). Here, we'll know out of band what is the kind of key (RSA or ECDSA) from another source, so this is not necessary (that means that the `UNKNOWN` handling is unnecessary).

In other words, we don't have to detect, we know when to call `RSA.parse` or `ECDSA.parse`.